javascript/javascript3/week3/canvas/index.js

```diff
@@ -0,0 +1,85 @@
+// Paint a circle to a canvas element
+/* function drawCircle(){
+    const canvas = document.getElementById('myCanvas');
+    if(canvas.getContext){
+        const ctx = canvas.getContext('2d');
+        ctx.beginPath();
+        ctx.arc(95, 50, 40, 0, 2 * Math.PI);
+        ctx.stroke();
+        ctx.fillStyle = "blue";
+        ctx.fill(); 
+        
+    }   
+}
+drawCircle(); */
+
+//Class creation time!
```

Comment:
There are a lot of commented code in the homework. If you are not using the code, then delete it.

Other than that, this class looks clean

Reply:
Ok sure